include/openssl/ossl_typ.h

```diff
@@ -177,6 +179,17 @@ typedef struct ct_policy_eval_ctx_st CT_POLICY_EVAL_CTX;
 typedef struct ossl_store_info_st OSSL_STORE_INFO;
 typedef struct ossl_store_search_st OSSL_STORE_SEARCH;
 
+typedef struct ESS_issuer_serial ESS_ISSUER_SERIAL;
+typedef struct ESS_cert_id ESS_CERT_ID;
+typedef struct ESS_signing_cert ESS_SIGNING_CERT;
+
+DEFINE_STACK_OF(ESS_CERT_ID)
```

Comment:
I don't see any other DEFINE_STACK_OF in this file, so I bet it is not its right place.

Reply:
That's correct, this header file contains only forward declarations by design.